zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementService.kt

```diff
@@ -98,16 +104,191 @@ class ObjectManagementService(
         )
 
         val objectsListDto = objectsList.results.map {
-            ObjectsListRowDto(it.url.toString(), listOf(
-                ObjectsListRowDto.ObjectsListItemDto("objectUrl", it.url),
-                ObjectsListRowDto.ObjectsListItemDto("recordIndex", it.record.index),
+            ObjectsListRowDto(
+                it.url.toString(), listOf(
+                    ObjectsListRowDto.ObjectsListItemDto("objectUrl", it.url),
+                    ObjectsListRowDto.ObjectsListItemDto("recordIndex", it.record.index),
 
-            ))
+                    )
+            )
         }
 
         return PageImpl(objectsListDto, pageable, objectsList.count.toLong())
     }
 
+    fun getObjectsWithSearchParams(
+        searchWithConfigRequest: SearchWithConfigRequest,
+        id: UUID,
+        pageable: Pageable
+    ): PageImpl<ObjectsListRowDto> {
+        val objectManagement = getById(id) ?: let {
+            logger.info {
+                "The requested Id is not configured as a objectnamagement configuration. " +
+                    "The requested id was: $id"
+            }
+            throw NotFoundException()
+        }
+
+        val searchFieldList = searchFieldV2Service.findAllByOwnerId(id.toString())!!
+
+        val searchDtoList = mutableListOf<ObjectSearchParameter>()
```

Comment:
You don't need this variable if the code below was making better use of the stream capabilities (using .map {} )